utils/utils_test.go

```diff
@@ -0,0 +1,90 @@
+package utils
+
+import "testing"
+
+func TestStripGithubPrefix(t *testing.T) {
+	tests := []struct {
+		name     string
+		input    string
+		expected string
+	}{
+		{
```

Comment:
Is it worth adding a test case for if there's no `github:` prefix? Or a malformed one?

Reply:
not sure if thats out of scope, maybe we should be doing a conftest for user PRs on that though?